ios/Runner/Info-Debug-prod.plist

```diff
@@ -2,8 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>ITSAppUsesNonExemptEncryption</key>
```

Comment:
curious why you removed this

Reply:
Xcode just moved it down below to line 50. Didn't manually edit this file